### PR TITLE
Adds Bubble tea to the game / Fixes drinks name

### PR DESCRIPTION
**Pull request by ClownMoff**
## About The Pull Request

Adds a new drink to the coffee vendor and capitalizes the R in "Robust coffee" in the process.

## Why It's Good For The Game

White women need more representation. Also it was requested here: 
https://forums.beestation13.com/t/add-bubble-tea/24274/20

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

imagen

https://github.com/BeeStation/BeeStation-Hornet/assets/148357670/6c47d055-b585-414b-8043-9ed5ed99b25c

</details>

## Changelog
:cl:Varo
add: Added a new drink to the coffee vendor: Bubble tea.
fix: Capitalized the R in "Robust coffee"
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 1]
```diff
@@ -1019,3 +1019,14 @@
 	M.adjust_disgust(30)
 	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "quality_drink", /datum/mood_event/quality_bad)
 	. = ..()
+
+/datum/reagent/consumable/bubble_tea
+	name = "Bubble Tea"
+	description = "Refreshing! You aren't sure what those things in the bottom are."
+	color = "#dfc7ab"
+	chem_flags = CHEMICAL_BASIC_DRINK
```
Comment (EvilDragonfiend) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]:
```suggestion
	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_BOTANY | CHEMICAL_GOAL_BARTENDER_SERVING
```
It isn't a basic drink if you can't get it from bartender's chem machine.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -1019,3 +1019,14 @@
 	M.adjust_disgust(30)
 	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "quality_drink", /datum/mood_event/quality_bad)
 	. = ..()
+
+/datum/reagent/consumable/bubble_tea
+	name = "Bubble Tea"
+	description = "Refreshing! You aren't sure what those things in the bottom are."
+	color = "#dfc7ab"
```
Comment (EvilDragonfiend) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: capitalize please

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]
I would want to see some effects on this, but it's fine anyway even if it has no special effect.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 2 above)
Comment (ClownMoff) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: I capitalize what?

**Comment by ClownMoff** [comment]
Regular tea doesn't have any effect AFAIC, idk what should I give the Bubbtle tea that isn't making it coffee 2.0

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 2 above)
Comment (EvilDragonfiend) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: DFC7AB

**Comment by EvilDragonfiend** [comment]
> Regular tea doesn't have any effect AFAIC, idk what should I give the Bubbtle tea that isn't making it coffee 2.0

If you don't have any special idea, it's okay. It's not that big deal.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 2 above)
Comment (ClownMoff) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Done

**Review by EvilDragonfiend — APPROVED** [review]

**Pull request merged**